presto-verifier/src/main/java/com/facebook/presto/verifier/event/QueryInfo.java

```diff
@@ -67,7 +68,8 @@ public QueryInfo(
             Optional<Double> cpuTimeSecs,
             Optional<Double> wallTimeSecs,
             Optional<Long> peakTotalMemoryBytes,
-            Optional<Long> peakTaskTotalMemoryBytes)
+            Optional<Long> peakTaskTotalMemoryBytes,
```

Comment:
wow. why not using `OptionalLong/OptionalDouble`?  -- this is irrelevant to this PR though

Reply:
Since we have everything in the `QueryStatsEvent`, we'll eventually deprecate those fields.